### ISSUE TITLE
WLSDPLY-05019 on MailSession

### ISSUE THREAD

**Issue by belfo** [oracle/weblogic-deploy-tooling#1118]
Hello,

i got a strange error today when "updating" an existing domain.

1. WLSDPLY-05019: Expected value of the mail.from property at location model_folders = ['MailSession'], 'name_tokens' = {'MAILSESSION': 'Mail Session'} to be a unicode data type, but it was a <type 'unicode'>
2. WLSDPLY-05019: Expected value of the mail.smtp.host property at location model_folders = ['MailSession'], 'name_tokens' = {'MAILSESSION': 'Mail Session'} to be a unicode data type, but it was a <type 'unicode'>

If i delete / recreate it works.
But the update fail.

Value are not changed.

What could cause this issue?

I'm using WebLogic Deploy Tooling 2.1.1

**Comment by CarolynRountree** [comment]
I am not able to recreate this scenario. Can you provide the model snippet via email at carolyn.rountree@oracle.com please? Also in the create, did you create the mail session and then update it?

**Comment by robertpatrick** [comment]
@belfo I also cannot replicate this issue using the following model:

```
topology:
    Server:
        AdminServer:
resources:
    MailSession:
        MyMailSession:
            JNDIName: mail/myMailSession
            SessionUsername: rpatrick
            SessionPasswordEncrypted: welcome1
            Properties:
                mail.smtp.host: my.mail.server.com
                mail.to: 'derek.sharpe@server.com'
            Target:  AdminServer
```

The Validate Model tool shows:

```
Issue Log for validateModel version 2.1.1 running WebLogic version 12.2.1.4.0 offline mode:

Total:       WARNING :     0    SEVERE :     0

validateModel.sh completed successfully (exit code = 0)
/Applications/WebLogic Kubernetes Toolkit UI.app/Contents/tools/weblogic-deploy/bin/validateModel.sh exited with exit code 0
```

Please let us know how to replicate your issue.

**Comment by robertpatrick** [comment]
This fix will be available in WDT 2.2.0. which we plan to release early next week.

**Comment by belfo** [comment]
thx :)
